nextflow

```diff
@@ -15,9 +15,17 @@
 #  limitations under the License.
 
 [[ "$NXF_DEBUG" == 'x' ]] && set -x
+
+home=$HOME
+if [[ "$HOME" == "/cygdrive/"* ]]; then
+  drive_letter="${HOME:10:1}"
+  replacement="$drive_letter:"
+  home="${HOME/\/cygdrive\/$drive_letter/$replacement}"
+fi
+
 NXF_VER=${NXF_VER:-'25.07.0-edge'}
 NXF_ORG=${NXF_ORG:-'nextflow-io'}
-NXF_HOME=${NXF_HOME:-$HOME/.nextflow}
+NXF_HOME=${NXF_HOME:-$home/.nextflow}
```

Comment:
Why using `home` instead of `HOME`? think this will break default launcher

Reply:
I see. indeed it would be preferable to use a different name. what about using `NXF_HOME` (if not defined yet)?